docs/source/configuration/authorization.mdx

```diff
@@ -0,0 +1,302 @@
+---
+title: Authorization in the Apollo Router
+---
+
+> ⚠️ **This is an [Enterprise feature](../enterprise-features/) of the Apollo Router.** It requires an organization with a [GraphOS Enterprise plan](https://www.apollographql.com/pricing/).
+>
+> If your organization _doesn't_ currently have an Enterprise plan, you can test out this functionality by signing up for a free [Enterprise trial](/graphos/org/plans/#enterprise-trials).
+
+The Apollo Router supports graph based authorization policies, through the `@authenticated` and `@requiresScopes` directives. They are used to authorize access to specific types or fields.
+
+They are defined as follows:
+
+```graphql
+directive @authenticated on OBJECT | FIELD_DEFINITION | INTERFACE | SCALAR | ENUM
```

Comment:
Should these be imported from a `@link`?

Reply:
probably yes. We'll update following @trevor-scheer's recommendations

Reply:
Yes:
```graphql
@link(url: "https://specs.apollo.dev/federation/v2.5", import: [..., "@authenticated", "@requiresScopes"])
```